src/Components/DoctorPage/Container/Patients.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Seems that DoctorPage should not be on shared 'components'.

Reply:
I create a folder named pages and move it to